tests/vmi_lifecycle_test.go

```diff
@@ -1428,3 +1517,32 @@ func nowAsJSONWithOffset(offset time.Duration) string {
 	Expect(err).ToNot(HaveOccurred(), "Should marshal to json")
 	return strings.Trim(string(data), `"`)
 }
+
+func syncKubevirtConfigMap() (cache.SharedIndexInformer, error) {
+	kubeClient, err := kubecli.GetKubevirtClient()
+	if err != nil {
+		return nil, err
+	}
+	kubeInformerFactory := controller.NewKubeInformerFactory(kubeClient.RestClient(), kubeClient, namespaceKubevirt)
```

Comment:
This doesn't guarantee us that the virt-api component actually received the informer update. 

As bad as this sounds. I'd suggest setting the configmap, and then sleeping 5 seconds before posting the vmi. There's really no practical way to know when the virt-api actually got the informer update.